tests/Feature/Controllers/ClientCreatePermissionCacheTest.php

```
@@ -34,31 +34,41 @@ protected function setUp(): void
+        $response2->assertStatus(200);
+        $response2->assertSee('Create New Client');
         $response3->assertStatus(200);
+        $response3->assertSee('Create New Client');
     }
 
     #[Test]
     public function it_allows_owner_to_access_clients_create_with_cached_permissions()
     {
+        /* Arrange */
         // Verify that cachedPermissions() returns usable Permission objects
         $owner = Role::where('name', 'owner')->first();
```

PHPStan (2.2.5)

[warning] Call to an undefined static method App\Models\Role::where().


(staticMethod.notFound)